hwilib/commands.py

```diff
@@ -217,6 +217,9 @@ def process_commands(args):
 
     signtransaction_parser = subparsers.add_parser('signtransaction', help='Sign a PSBT')
     signtransaction_parser.add_argument('--psbt', help='The Partially Signed Bitcoin Transaction to sign')
+    signtransaction_parser.add_argument('--testnet', help='Use testnet prefixes', action='store_true')
+    signtransaction_parser.add_argument('--fingerprint', help='Specify the device to connect to using the first 4 bytes of the hash160 of the master public key. It will connect to the first device that matches this fingerprint.')
```

Comment:
Why? `--fingerprint` is a global option.

Reply:
What makes something global?

Reply:
I removed the commit that adds this redundancy.